Packs/Intezer/Integrations/IntezerV2/README.md

```diff
@@ -688,3 +698,94 @@ size_in_bytes	        245760
 ssdeep	                ssdeep
 ```
 
+### intezer-get-analysis-iocs
+
+***
+Gets the list of network and files IOCs of a specific analysis id.
```

Comment:
I think that this is a better description than the one you added to the YML - worth adding that one to the YML file instead.